home/models.py

```diff
@@ -277,15 +292,12 @@ class WhatsappBlock(blocks.StructBlock):
         help_text="List title, up to 24 characters.",
         max_length=24,
     )
-    list_items = blocks.ListBlock(
-        blocks.CharBlock(label="Title"),
-        default=[],
+    list_items = blocks.StreamBlock(
+        [("next_message", NextMessageListItem()), ("go_to_page", GoToPageListItem())],
         help_text="List item title, up to 24 characters.",
```

Comment:
We should probably change this help text as well

Reply:
What should it be? "List item text, up to 24 characters"?

Reply:
Well this is now just the items in the list right? It's no longer just the text, each text block will have their own help text mentioning the text and character limit. Maybe "Items to appear in the list message"

Reply:
ohhh, right, with you now